code/game/objects/items/grenades/flashbang.dm

```diff
@@ -50,6 +51,22 @@
 			living_mob.Knockdown(30)
 		living_mob.soundbang_act(1, max(200 / max(1, distance), 60), rand(0, 5))
 
+/obj/item/grenade/flashbang/change_det_time(time) //Time uses real time.
+	. = TRUE
+	if(!isnull(time))
+		det_time = round(clamp(time * 10, 3 SECONDS, 5 SECONDS))
+	else
+		var/previous_time = det_time
+		switch(det_time)
```

Comment:
Instead of this override would it be possible to rewrite the base proc to also use `possible_fuse_time`?

Reply:
Yeah probably but it would require re-making screwdriver_act() from scratch because it's a hard coded switch